### PR TITLE
Fix for flakey time-dependent test

### PR DESCRIPTION
**Pull request by stevehook**
## Context

This test failed late in the evening with an off by one 1 day error when
asserting a date output. I think this is because the test takes a
finite amount of time to complete it's steps and I got unlucky with the
assertion happening the day after the code under test ran. So a bit of
an edge case. 

## Changes proposed in this pull request

I think the only reliable way to fix this is with `Timecop.freeze` rather than `travel`. Open to any other ideas.

## Guidance to review

Is there a better way to fix this that avoids `Timecop.freeze`?

## Link to Trello card

n/a

## Things to check

- [x] This code does not rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] This code does not rely on the addition/removal of Azure config environment variables in the same Pull Request
- [x] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/docs/environment-variables.md#azure-hosting-devops-pipeline)

### CODE REVIEW

File: spec/system/candidate_interface/references/candidate_views_reference_history_spec.rb [hunk 1]
```diff
@@ -29,22 +33,22 @@ def and_i_add_a_reference
...
-    Timecop.travel(@reference.requested_at + 1.day) do
+    Timecop.freeze(@reference.requested_at + 1.day) do
       click_link 'Send a reminder to this referee'
       click_button 'Yes I’m sure - send a reminder'
     end
   end
 
   def and_the_system_sends_an_automated_reminder
-    Timecop.travel(@reference.requested_at + TimeLimitConfig.chase_referee_by.days) do
+    Timecop.freeze(@reference.requested_at + TimeLimitConfig.chase_referee_by.days + 1.minute) do
```
Comment (stevehook) [inline spec/system/candidate_interface/references/candidate_views_reference_history_spec.rb:51]: The plus one minute is needed here because `ChaseReferences` checks for time now being greater than (not greater than or equal) to the time limit.

**Review by davidgisbey — APPROVED** [review]
